src/components/company-form/company-form.js

```diff
@@ -2,68 +2,123 @@ import React, { Component } from 'react';
 import Button from 'react-bootstrap/Button'
 import 'bootstrap/dist/css/bootstrap.min.css';
 
+import './company-form.css'
 
 export class CompanyForm extends Component{
     constructor(props) {
         super(props);
-        this.isEdit = !!props.company;
+        this.isEdit = !props.date;
```

Comment:
1. add `company` in props instead of `name`, `address` etc.
2. in this case it will be `this.isEdit = !props.company;`

Reply:
done